app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -95,11 +97,25 @@ private void addBuildingMarkers() {
         for (Building building : loyBuildings) {
             createBuildingMarkersAndPolygonOverlay(building);
         }
+        map.setOnPolygonClickListener(new GoogleMap.OnPolygonClickListener() {
+            @Override
+            public void onPolygonClick(Polygon polygon) {
+                ((MainActivity) getActivity()).createToast("Building Clicked");
```

Comment:
It was not for debugging purpose. I put this code there so people can know how to read a click event from a building. The toast should be replaced by the building popup UI.